### PR TITLE
Prep for 1.1.5 release

### PR DESCRIPTION
**Pull request by BryanEllington**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+    the updated dashboards to replace the prior versions.
+
+* **Logging**
+  * [FEATURE] - Enhanced multi-line support is now enabled by default in
+    Fluent Bit
+  * [FIX] - Several status check  in scripts have been simplified to use
+    `kubectl wait`
+  * [FIX] - All logging components now specify Kubernetes resource requests
+  * [CHANGE] - Event router is now deployed to the `logging` namespace instead
+    of the `kube-system` namespace
```
Comment (cumcke) [inline CHANGELOG.md:28]: This change was done in the previous version (1.1.3).  This sprint, we fixed it to where Event Router with Work Node Placement enabled now correctly gets deployed in the logging namespace.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
+  * [DOC] - Documentation on log retention has been reviewed and re-worked to
+    be more clear and correct
```
Comment (gsmith-sas) [inline CHANGELOG.md:30]: [DOC] - The [documentation on how to adjust log retention](logging/LogRetention.md) has been revised to improve clarity and correct errors.  Fixes #261.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,47 @@
...
+  * [CHANGE] - The deprecated `KB_TLS_ENABLE` flag has been removed. Kibana TLS
+    is now set via the normal `TLS_ENABLE` and `LOG_TLS_ENABLE` flags
```
Comment (gsmith-sas) [inline CHANGELOG.md:32]: change "...is now set via..." to "...is now controlled via..."

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,47 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.1.5 (15FEB22)
+
+* **Overall**
+  * [FIX] Several spurious error messages have been suppressed
```
Comment (gsmith-sas) [inline CHANGELOG.md:6]: Which "several spurious" error messages?  If this is related to the errexit trap, we identified a single spurious message...which we addressed by fixing the code which was generating an error.

Comment (BryanEllington) [reply CHANGELOG.md:6]: I thought there was more than one case where this came up. Could you provide more accurate wording?

**Review by gsmith-sas — APPROVED** [review]

**Review by cumcke — APPROVED** [review]
Looks good to me.

**Pull request merged**